### PR TITLE
Support using drawable as Android small icon

### LINKED ISSUE #1594: [Android] Can smallIcon is only from mipmap?
# Question

At first, thanks to repo, I implemented my notification feature.
But, I have one question,

I generated `notification icon` assets using android studio.
`notification_icon` was automatically saved in `drawable` not `mipmap`, but in `react-native-push-notification/android/src` `RNPushNotificationHelper.java`

```
/// Line 356

if (smallIcon != null && !smallIcon.isEmpty()) {
  smallIconResId = res.getIdentifier(smallIcon, "mipmap", packageName);
} else if(smallIcon == null) {
  smallIconResId = res.getIdentifier("ic_notification", "mipmap", packageName);
}

if (smallIconResId == 0) {
    smallIconResId = res.getIdentifier("ic_launcher", "mipmap", packageName);

    if (smallIconResId == 0) {
        smallIconResId = android.R.drawable.ic_dialog_info;
    }
}

notification.setSmallIcon(smallIconResId);

```

Only assign image in `mipmap`, so how about fix it to use both `mipmap` and `drawable` for `smallIcon` ?

### PR DESCRIPTION
**Pull request by hojason117**
Add support for using drawable resource as Android local notification small icon.
Default using drawable since Image Asset Studio puts notification icons in there.

fixes #1594

### CODE REVIEW

File: android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java [hunk 1]
```diff
@@ -341,9 +344,9 @@ public void sendToNotificationCentreWithPicture(Bundle bundle, Bitmap largeIconB
                 String largeIcon = bundle.getString("largeIcon");
 
                 if (largeIcon != null && !largeIcon.isEmpty()) {
-                  largeIconResId = res.getIdentifier(largeIcon, "mipmap", packageName);
+                    largeIconResId = res.getIdentifier(largeIcon, "mipmap", packageName);
```
Comment (Dallas62) [inline android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java:347]: This should also include the same kind of condition, like it was made with `smallIcon` to get drawable.

**Review by Dallas62 — COMMENTED** [review]
Hi @hojason117 

Thanks for this PR, there is only one change missing to be merged 😉

File: android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java (hunk 1 above)
Comment (hojason117) [reply android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotificationHelper.java]: Same condition for large icon added 👍

**Review by Dallas62 — APPROVED** [review]

**Comment by Dallas62** [comment]
Thanks for this PR !

**Pull request merged**